### ISSUE TITLE
Phoenix Wright: Ace Attorney - Dual Destinies – Fail to boot if system titles update installed (Regression)

### ISSUE THREAD

**Issue by Dragios** [citra-emu/citra#5141]
*PW:AA - DD* fails to boot with ```unmapped Read32``` error after #4913 is merged to Nightly. The game still boots fine if system titles update is absent. Before **Core Timing 2.0 PR**, it boots irregardless of whether system titles is present or absent.

This is the error:
```
HW.Memory <Error> core/memory.cpp:Read:181: unmapped Read32 @ 0x00000000
HW.Memory <Error> core/memory.cpp:Read:181: unmapped Read32 @ 0x00000004
HW.Memory <Error> core/memory.cpp:Read:181: unmapped Read32 @ 0x00000008
HW.Memory <Error> core/memory.cpp:Read:181: unmapped Read32 @ 0x0000000C
```
This error log will spam the same thing except the memory location will increase infinitely with +4
Full log: [citra_log.txt](https://github.com/citra-emu/citra/files/4373562/citra_log.txt)

@xperia64 has confirmed this issue which means it is reproducible.
cc @B3n30

**Comment by Dragios** [comment]
Seems like Core Timing 2.0 also breaks other LLE applets from launching. I can no longer trigger software keyboard on Mii Maker. Will result in unmapped Read32.
Most likely this is the same reason PW:AA - Dual Destinies failed to boot. It failed to launch the eshop applet.

**Comment by Dragios** [comment]
Fixed with PR #5193
